renovate.json5

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:base"
+  ],
+  "timezone": "Asia/Tokyo",
```

Comment:
diff may not work but changes are just
1. rename renovate.json to renovate.json5 to write comment
2. add `timezone` and `packageRules[].schedule`